packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -52,6 +55,14 @@ export const startOptions: CliCommandOptions<StartArgs> = {
       "The trusted checkpoint root to start the lightclient. If not provided will initialize from the latest finalized slot. It shouldn't be older than weak subjectivity period",
     type: "string",
   },
+
+  "unverified-whitelist": {
```

Comment:
On the casing topic, almost all of the CLI flags on Lodestar are using camelCase. For the prover to have same look and feel would probably make sense to align this.

Reply:
I updated and renamed all cli options for the prover to camelCase.